<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
      <item>
       <widget class="QComboBox" name="bookSelector">
        <item>
         <property name="text">
          <string>Book 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Book 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Book 3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Book 4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Book 5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Book 6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Book 7</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="pageSelLayout">
        <item>
         <widget class="QLabel" name="pageLabel">
          <property name="text">
           <string>Page</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pageInput"/>
        </item>
        <item>
         <widget class="QCheckBox" name="addCheckBox">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="listView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="delButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFlands_Helper">
    <property name="title">
     <string>Flands Helper</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFlands_Helper"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as..</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="2" margin="4"/>
 <resources/>
 <connections/>
</ui>
